lib/tool_shed/util/hg_util.py

```diff
@@ -395,21 +396,21 @@ def init_repository(repo_path):
     try:
         subprocess.check_output(['hg', 'init'], stderr=subprocess.STDOUT, cwd=repo_path)
     except Exception as e:
-        error_message = 'Error initializing repository: %s' % e
+        error_message = 'Error initializing repository: %s' % unicodify(e)
         if isinstance(e, subprocess.CalledProcessError):
-            error_message += "\nOutput was:\n%s" % e.output
+            error_message += "\nOutput was:\n%s" % unicodify(e.output)
         raise Exception(error_message)
 
 
 def changeset2rev(repo_path, changeset_revision):
     """
-    Return the revision number corresponding to a specified changeset revision.
+    Return the numeric revision number (as a string) corresponding to a specified changeset revision.
```

Comment:
Numeric number doesn't read well
```suggestion
    Return the revision number (as a string) corresponding to a specified changeset revision.
```